templates/docs/examples/patterns/forms/radio-validation.html

```diff
@@ -0,0 +1,16 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Radio Validation{% endblock %}
+
+{% block standalone_css %}patterns_form-tick-elements{% endblock %}
+
+{% block content %}
+<form>
+    <div class="p-form-validation is-error">
+      <label class="p-radio is-required">
+        <input type="radio" class="p-radio__input" name="radioPattern" required aria-labelledby="radioExample4">
+        <span class="p-radio__label" id="radioExample4">is-required *</span>
```

Comment:
Label with class "is-required" adds the * via CSS so there is no need for it in text.

```suggestion
        <span class="p-radio__label" id="radioExample4">is-required</span>
```